Description: Fix for -Werror=implicit-function-declaration errors
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1066338
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2024-03-13
Forwarded: no


--- a/interpolation/emos.h
+++ b/interpolation/emos.h
@@ -20,6 +20,8 @@
 #define     RSECTION_3  2
 #define     RSECTION_4  1
 
+void jnumgg_(const fortint *knum_, const char *htype_, int kpts_[], fortint *kret_);
+fortint int2_intuvu(fortfloat vort_in[],fortfloat div_in[] , fortint in_array_length, fortfloat *vort_out, fortfloat *div_out, fortint *out_array_length);
 
 fortint int2_global();
 fortint outrep_();
